src/@next/Switch/SwitchStyle.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import { Breakpoints, Spacing } from '..';
+
+import { Blue, Neutral } from '../utilities/colors';
+import { SwitchProps } from './Switch';
+
+export const SwitchStyle = styled.div<Pick<SwitchProps, 'disabled'>>`
```

Comment:
And if you can, you will need to remove this Typing

Reply:
removed